docs: v5.8.0 (#43925)
ant-design/ant-design: CHANGELOG.en-US.md

---

## 5.8.0

`2023-08-01`

> - 🆕 Descriptions implement `items` API. [#43483](https://github.com/ant-design/ant-design/pull/43483) [@RedJue](https://github.com/RedJue)

Reviewer: 一般风格都是 `support xxx prop`

- 🆕 ColorPicker implement `disabledAlpha` API. [#43355](https://github.com/ant-design/ant-design/pull/43355) [@RedJue](https://github.com/RedJue)

> - 🆕 Modal hooks function support `await` call. [#43470](https://github.com/ant-design/ant-design/pull/43470)

Reviewer: 这个换 🔥 图标，放最上面~

- 🆕 Avatar.Group support `shape` props. [#43817](https://github.com/ant-design/ant-design/pull/43817) [@li-jia-nan](https://github.com/li-jia-nan)

> - 🆕 AutoComplete/Cascader/DatePicker/Input.Textarea/TimePicker/TreeSelect support `allowClear` prop to customize clear button。[#43582](https://github.com/ant-design/ant-design/discussions/43582) [@kiner-tang](https://github.com/kiner-tang)

Reviewer: 结尾标点错了

> - 🆕 AutoComplete/Cascader/DatePicker/Input/Input.Textarea/TimePicker/TreeSelect support `allowClear` prop to customize clear button。[#43582](https://github.com/ant-design/ant-design/discussions/43582) [@kiner-tang](https://github.com/kiner-tang)

Reviewer 2: Input 以前就支持的，不是这次加的
```suggestion
- 🆕 AutoComplete/Cascader/DatePicker/Input.Textarea/TimePicker/TreeSelect support `allowClear` prop to customize clear button。[#43582](https://github.com/ant-design/ant-design/discussions/43582) [@kiner-tang](https://github.com/kiner-tang)
```

> - 🆕 Component Token support `algorithm` to calculate derivative tokens same as global. [#43810](https://github.com/ant-design/ant-design/pull/43810) [@MadCcc](https://github.com/MadCcc)

Reviewer: 🔥 图标，移动到最上面

> - 🆕 Form support `getFieldsValue({ strict: true })` to support only Item bind values. [#43828](https://github.com/ant-design/ant-design/pull/43828)

Reviewer: 不用 火，但是移动到最上面~

- 🆕 RangePicker `presets` support callback functions. [#43476](https://github.com/ant-design/ant-design/pull/43476) [@Wxh16144](https://github.com/Wxh16144)
- 🆕 Image `PreviewType` support `movable`. [#43823](https://github.com/ant-design/ant-design/pull/43823) [@linxianxi](https://github.com/linxianxi)
- 🆕 Slider `tooltip` support `autoAdjustOverflow` prop. [#43788](https://github.com/ant-design/ant-design/pull/43788)

> - 🆕 ConfigProvider support `wave` to customize wave effect. [#43784](https://github.com/

Reviewer: 🔥 同上

> - 🆕 ConfigProvider support `wave` to customize wave effect. [#43784](https://github.com/

Reviewer 2: 少个括号
```suggestion
- 🆕 ConfigProvider support `wave` to customize wave effect. [#43784](https://github.com/)
```

- 🆕 Added the `selectionsIcon` property to the Transfer component to support custom icons for the dropdown menu. [#43773](https://github.com/ant-design/ant-design/pull/43773) [@li-jia-nan](https://github.com/li-jia-nan)

> - 🗑 Select, Tree-Select and Cascader component deprecated `showArrow` prop. [#43520](https://github.com/ant-design/ant-design/pull/43520) [@MuxinFeng](https://github.com/MuxinFeng)

Reviewer 3: ```suggestion
- 🗑 Select, Tree-Select and Cascader deprecated `showArrow` prop. Now suffix arrow should be configured with `suffixIcon`. [#43520](https://github.com/ant-design/ant-design/pull/43520) [@MuxinFeng](https://github.com/MuxinFeng)
```

- 🐞 Fix `Modal.confirm` `locale` setting were reset. [#43277](https://github.com/ant-design/ant-design/pull/43277) [@Yuiai01](https://github.com/Yuiai01)
- 🐞 Fix Slide description info and slider handle overlap issue. [#43780](https://github.com/ant-design/ant-design/pull/43780) [@Wxh16144](https://github.com/Wxh16144)

> - 🐞 Fix InputNumber handle style in large size. [#43875](https://github.com/ant-design/ant-design/pull/43875) [@yee94](https://github.com/yee94)

Reviewer 4: ```suggestion
- 🐞 Fix InputNumber handler style in large size. [#43875](https://github.com/ant-design/ant-design/pull/43875) [@yee94](https://github.com/yee94)
```
